src/index.web.tsx

```diff
@@ -0,0 +1,122 @@
+type Profiler = any;
```

Comment:
hm, so we type "Profiler" as any 🤔  can we create a type for this here please? Or maybe we find some type online we can use